### DISCUSSION TITLE
How can I access the SDL_GL_Context with raylib compiled with the SDL backend?

### DISCUSSION THREAD

**Discussion by DeuzivanViana** [raysan5/raylib#4611] [Q&A]
Hello, I'm working on a project where I'm trying to use raylib's ease of programming along with SDL2's compatibility with other libraries (like ImGui). I need to access the SDL_GL_Context to work with the context, but there seems to be no way unless I edit rcore, from what I've tried. Is there a way, such as accessing PlatformData?

**Comment by raysan5** [comment]
@DeuzivanViana I'm afraid there is no way to access it without modifying raylib, it's not exposed by default. If the reason is DearImgui, you can check https://github.com/raylib-extras/rlImGui
